go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 require (
 	github.com/argoproj/argo-cd v0.8.1-0.20210218202601-6de3cf44a4cb
+	github.com/argoproj/gitops-engine v0.2.1-0.20210129183711-c5b7114c501f
```

Comment:
I think just because when I imported it, GoLand automagically used that version. We're just importing it for the constants in the `health` package, so I don't think it's a big deal but I'll see if we I can update it.